torch/fx/experimental/symbolic_shapes.py

```diff
@@ -155,6 +155,21 @@ def is_concrete_bool(a: Union[bool, SymBool]):
 
     return False
 
+def is_singleton(s):
+    # check for SingletonSymNode
+    if not isinstance(s, torch.SymInt):
+        return False
+    if s.node.singleton_int() is not None:
+        return True
+
+    # check for SymInt wrapping a SingletonSymNode (fake-ifying causes this)
```

Comment:
comment is inaccurate, it's not SymInt, it's a symbolic variable (since SymInt encapsulates both symbolic and not)